terraform/modules/emr/templates/emr/get_scripts.sh

```diff
@@ -7,6 +11,8 @@ SOURCE_PATH=$BUCKET/$SOURCE_LOCATION
 (
     # Import the logging functions
     source /opt/emr/logging.sh
+    
+    sudo mkdir -p $DESTINATION_LOCATION
```

Comment:
Looks good (if we create the directory using sudo though, wont it only be writeable by root?) Dont we need to chmod it for the copying to work, or will it have write permissions by default?